packages/material-ui/src/utils/reactHelpers.js

```diff
@@ -25,3 +25,18 @@ export function setRef(ref, value) {
     ref.current = value;
   }
 }
+
+export function useForkRef(refA, refB) {
```

Comment:
We have two schools here. I do ref1, ref2, you do refA, refB, it's not something I'm proud of 🙈😂. Should the method be variadic?

Reply:
> variadic

I though about this and I don't think there are legitimate use cases other then deprecation (see [ButtonBase.js](https://github.com/mui-org/material-ui/blob/0713bd27e7b8a4a64b49edf32d79925ef83abffd/packages/material-ui/src/ButtonBase/ButtonBase.js#L308). 

So far we only have use for this if we need our own ref and want to expose this. We can make this variadic if this happens more often but for now I would not overabstract this since it adds size and is problematic if the number of arguments changes.

Reply:
Where do you use `ref1` and `ref2`? It doesn't really matter what enumeration we use.

Reply:
It's more generic. When I have two variables with the same name, I would often use 1, 2, 3 instead of finding a better name. I'm just mentioning for context, A, B, C is great too.